lldb/source/API/SBProcessInfoList.cpp

```diff
@@ -0,0 +1,73 @@
+//===-- SBProcessInfoList.cpp -------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "lldb/API/SBProcessInfoList.h"
+#include "Utils.h"
```

Comment:
sort ?

Reply:
This is sorted by clang-format, it wants the corresponding header first, then sorter alphabetically. I'll move utils down to avoid this weird looking situation.